java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -395,6 +400,9 @@ protected void rampSpeedTo(String endSpeedType, int endBlockIdx, boolean useInde
                     time += 20;
                 }
                 catch (InterruptedException ie) { // ignore
```

Comment:
I suggest removing the `// ignore` comment since the exception is no longer ignored.